themes/bootstrap3/templates/search/results-list.phtml

```diff
@@ -20,4 +20,4 @@
     $this->resultsAttrs->add('data-history', $this->saveToHistory ? '1' : '0');
   }
 ?>
-<?=$this->render('search/list-' . $this->params->getView() . '.phtml')?>
+<?=$this->render('search/list-' . $this->params->getView() . '.phtml', ['results' => $this->results])?>
```

Comment:
If this is solving the problem, I think it's probably also creating other new problems. The list-* templates are designed to be rendered in the same scope as results-list.phtml, so if we only pass through the 'results' value, we're going to disable a number of overrides and configurations. I suspect we're going to need a more targeted solution for the problem you're seeing (and a better understanding of the cause). Might it be helpful to start with a test case that reproduces the issue? Also, have you talked to @EreMaijala about this problem? Since he's done much of the recent search refactoring and sid work, he might have more insight than I do on what's happening.

Reply:
Just asked and seems like it might be deeper somewhere, search continues!